src/guides/v2.3/install-gde/install/cli/install-cli-subcommands-maint.md

```diff
@@ -85,6 +85,43 @@ where
 
 `--none` clears the list.
 
+## For Multistore setups {#instgde-cli-multistore}
+
+If we want to setup multiple stores with different layout and localized content for each and every store, we have to use an out of the box feature to achieve it. That is by passing the `$_GET['skin']` parameter to the intended processor. For example, in a sample scenario, The error processor accepts an parameter to change the error page template. Here, we have considered a `503` type error template file for which we have provided the localized content. Below are the general steps to follow,
+
+```php
+if (isset($_GET['skin'])) {
+    $this->_setSkin($_GET['skin']);
+}
+```
+
+This can be passed via rewrite rule in .htaccess and needs no further changes.
+
+# For $_GET['skin'] parameter {#instgde-cli-multistore-getskin}
+
+*  Check if the `.maintenance.flag` exists.
+*  Check for the host (or any other form like the ENV variables)
```

Comment:
The second and third instruction are confusing as it's not clear what's being checked, where it's being checked and what gets set.